pyethapp/jsonrpc.py

```diff
@@ -839,10 +840,10 @@ class Filter(object):
     :ivar new_logs: same as :attr:`logs`, but is reset at every access
     """
 
-    def __init__(self, chain, blocks=[], addresses=None, topics=None,
+    def __init__(self, chain, blocks, addresses=None, topics=None,
```

Comment:
Yeah, my goal wasn't to change the signature, but to avoid the problematic mutable `[]`.